pkg/descheduler/strategies/lownodeutilization_test.go

```diff
@@ -514,6 +517,227 @@ func TestLowNodeUtilization(t *testing.T) {
 			// 0 pods available for eviction because there's no enough extended resource in node2
 			expectedPodsEvicted: 0,
 		},
+		{
+			name: "without priorities, but only other node is unschedulable",
+			thresholds: api.ResourceThresholds{
+				v1.ResourceCPU:  30,
+				v1.ResourcePods: 30,
+			},
+			targetThresholds: api.ResourceThresholds{
+				v1.ResourceCPU:  50,
+				v1.ResourcePods: 50,
+			},
+			nodes: map[string]*v1.Node{
+				n1NodeName: test.BuildTestNode(n1NodeName, 4000, 3000, 9, nil),
+				n3NodeName: test.BuildTestNode(n3NodeName, 4000, 3000, 10, test.SetNodeUnschedulable),
```

Comment:
rather n2NodeName otherwise my first Q was where's the second node.

Reply:
Fair point, fixed.